apps/src/templates/lessonOverview/activities/LevelDetailsDialog.jsx

```diff
@@ -74,7 +83,41 @@ export default class LevelDetailsDialog extends Component {
         </div>
       );
     } else {
-      return <div>Not implemented yet</div>;
+      // TODO: calculate more of these parameters based on the level and pages
+      return (
+        <UnconnectedTopInstructions
+          hasContainedLevels={false}
+          noVisualization={true}
+          isMinecraft={false}
+          isBlockly={false}
+          isRtl={false}
```

Comment:
Whats the plan with RTL? Is that dealt with in CSF instructions?

Reply:
I meant to handle it as a follow-up but I'm realizing it's pretty easy to include in this PR, so I'll update.